release/downloadIstioCandidate.sh

```diff
@@ -79,6 +79,10 @@ ARCH_URL="https://github.com/istio/istio/releases/download/${ISTIO_VERSION}/isti
 
 with_arch() {
   printf "\nDownloading %s from %s ...\n" "$NAME" "$ARCH_URL"
+  if ! curl --output /dev/null --silent --head --fail "$ARCH_URL"; then
+    printf "\n%s is not found, please specify right ISTIO_VERSION and TARGET_ARCH\n" "$ARCH_URL"
```

Comment:
Maybe change the message:
```suggestion
    printf "\n%s is not found, please specify a valid ISTIO_VERSION and TARGET_ARCH\n" "$ARCH_URL"
```